src/emulator/helpRVL.c

```diff
@@ -216,6 +195,62 @@ static void fn_8005E638(GXColor color) {
     GXSetTevColor(GX_TEVREG0, color);
 }
 
+#pragma inline_max_auto_size(100)
+
+void helpMenuSetupRender(GXTexObj* pTexObj) {
+    Rect rect = {0};
+    GXColor color;
+    f32 fWidth;
+    f32 fHeight;
+    Mtx44 matrix44;
+    f32 x1 = 0.0f;
+    f32 y1 = 0.0f;
+    f32 y0 = 0.0f;
+    f32 x0 = 0.0f;
+
+    rect.x1 = sRenderMode->fbWidth / 2;
+    rect.y1 = sRenderMode->xfbHeight / 2;
+
+    GXInvalidateVtxCache();
+    GXInvalidateTexAll();
+
+    if (fn_8007FC84()) {
+        fWidth = ((sRenderMode->viWidth - 704) * 320) / 1408.0f;
```

Comment:
here too it would probably make sense with the macros but idk, I noticed that yesterday when I tried but I wasn't sure